graphql_api/tests/test_owner.py

```diff
@@ -569,3 +569,21 @@ def test_owner_query_with_no_service(self):
 
         assert res["errors"][0]["message"] == MissingService.message
         assert res["data"]["owner"] is None
+
+    def test_owner_query_with_private_repos(self):
```

Comment:
might be worth it to test if owner doesn't have any private repos

Reply:
Done!